.github/workflows/build_test_deploy.yml

```diff
@@ -1537,7 +1537,7 @@ jobs:
         run: node scripts/normalize-version-bump.js
 
       - name: Build
-        run: turbo run build-wasm -- --target ${{ matrix.target }}
+        run: turbo run build-wasm -- --target ${{ matrix.target }} --features tracing/release_max_level_info
```

Comment:
Should we add a `build-wasm-release` script and use that?

Reply:
Not sure, do you prefer it?